pkg/dcmtime/parse.go

```diff
@@ -0,0 +1,376 @@
+package dcmtime
+
```

Comment:
Still haven't had a chance to review this file

Reply:
I've broken this file up, the parse functions now reside in their respective type's files.